src/frontend/components/PreviewTable/index.js

```diff
@@ -57,6 +57,27 @@ const EditButton = styled.button`
   }
 `;
 
+const SummaryBtn = styled.button`
```

Comment:
We should already have a Button component with adjustable features that matches what you're trying to do here; you can find that under components/Button.